pandas/tests/groupby/test_timegrouper.py

```diff
@@ -887,9 +887,7 @@ def test_groupby_apply_timegrouper_with_nat_apply_squeeze(
         # We need to create a GroupBy object with only one non-NaT group,
         #  so use a huge freq so that all non-NaT dates will be grouped together
         tdg = Grouper(key="Date", freq="100Y")
-
-        with tm.assert_produces_warning(FutureWarning, match="`squeeze` parameter"):
-            gb = df.groupby(tdg, squeeze=True)
+        gb = df.groupby(tdg)
```

Comment:
I think you can just remove this entire test as it was intentionally testing the squeeze functionality

Reply:
Yeah true, but this shows nicely how it changed in cases where squeeze had an actual effect. I think this would be nice in our history if someone wants to go back and have a look